sources/packages/backend/libs/services/src/sfas/sfas-application.service.ts

```diff
@@ -73,6 +75,48 @@ export class SFASApplicationService extends DataModelService<SFASApplication> {
     return +(total?.sum ?? 0);
   }
 
+  /**
+   * Fetch the SFAS full time application for the student and the latest
+   * application which has the end date within 730 days.
+   * @param studentId student id.
+   * @returns SFASSignedMSFAA which contains the SFAS Signed MSFAA
+   * and latest application end date.
+   */
+  async getIndividualFullTimeApplicationByIndividualId(
+    studentId: number,
+  ): Promise<SFASSignedMSFAA> {
+    const twoYearsAgo = new Date();
+    twoYearsAgo.setDate(twoYearsAgo.getDate() - MAX_MSFAA_VALID_DAYS);
+    const [sfasApplication] = await this.repo.find({
+      select: {
+        id: true,
+        endDate: true,
+        individual: {
+          id: true,
+          msfaaNumber: true,
+        },
+      },
+      relations: {
+        individual: true,
+      },
+      where: {
+        individual: { id: studentId },
```

Comment:
I am not sure if I am following this. Individual ID presents the SFAS ID. The table also has the `student_id` which is probably the where clause that we are looking for here. Does it sound right?